ui/app/AppLayouts/Profile/views/wallet/NetworksView.qml

```diff
@@ -101,6 +109,7 @@ Item {
                     checked: walletStore.areTestNetworksEnabled
```

Comment:
why doesnt this only work?

Reply:
wdym?

Reply:
what I mean is this is a binding in its own self, why are we adding a binding again?

Reply:
> why are we adding a binding again?

Cause clicking on a toggle changes the `checked` state internally and breaks the previous binding, also we want to change it when the setting is changed for real, which means after confirming the change in the popup.

Reply:
ohhh okay, get it now, looks good :)